drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -209,8 +210,22 @@ class DragAndDropBlock(
         enforce_type=True,
     )
 
+    raw_possible = Float(
```

Comment:
weight attribute already existed but was not being used? And how was score being recorded before raw_possible attribute?